source/Account.cpp

```diff
@@ -224,6 +224,19 @@ string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
 	// makes you at least as credit-worthy as someone who pays debts on time.)
 	creditScore = max(200, min(800, creditScore + (missedPayment ? -5 : 1)));
 	
+	auto creditString = [](int64_t payment) -> string
+	{
+		return payment == 1 ? "1 credit" : Format::Credits(payment) + " credits";
+	};
+	
+	if(maintenancePaid < 0)
+	{
+		if(missedPayment)
+			out << " ";
+		out << "You received " << creditString(-maintenancePaid) << " in maintenance.";
```

Comment:
Maintenance only makes money when you are the person doing the maintenance, which is not the case here.

Maybe reword so something like:
```suggestion
		out << "Your ships and outfits netted you " << creditString(-maintenancePaid) << ".";
```

Reply:
> Maintenance only makes money when you are the person doing the maintenance, which is not the case here.

Or reword to: "You received (credits) based on your ships and outfits."